verifier.py

```diff
@@ -355,31 +355,68 @@ def __init__(self):
         self.minimum_size = 80
         self.page_elements = [
             '#projects',
-            '#projects li',                              # Specific project list item
-            'body div.panel-body',                       # Component list
-            'body h2'                                    # Activity points, project count
+            '#projects li',  # Specific project list item
+            'body div.panel-body',  # Component list
+            'body h2'  # Activity points, project count
         ]
         self.harvest_pages(run_info['user_profile_page_urls'], '', UserProfilePage)
         self.size_comparison()
         self.spot_check()
 
 
-class InstitutionProfileVerifier(Verifier):
+class InstitutionDashboardVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
         self.minimum_size = 350
         self.page_elements = [
             '#fileBrowser > div.db-infobar > div > div',  # Project preview
-            '#tb-tbody'                                   # Project browser
+            '#tb-tbody'  # Project browser
         ]
         self.harvest_pages(run_info['institution_urls'], '', InstitutionProfilePage)
         self.size_comparison()
         self.spot_check()
 
 
-# Main execution
-
-RegistrationFiles = RegistrationFilesVerifier()
-
-print("Retry list: ", RegistrationFiles.failed_pages)
+def main():
+    # for modularization and then concatenate lists
+    rescrape_list = []
+    for i in range(NUM_RETRIES):
```

Comment:
This whole block should be inside `if run_info['scrape_finished']:` so we don't risk overstepping the resume functionality